### PR TITLE
cms: create “oauth2-api-ory-cloud-update”

in Blog post

### PR DESCRIPTION
**Pull request by zeropaper**
Automatically generated by Netlify CMS

### CODE REVIEW

**Review by BlueAcidFrog — APPROVED** [review]

**Comment by BlueAcidFrog** [comment]
Sorry, i was too quick. I can't see the blog post: 

Bildschirmfoto 2022-01-21 um 15 05 53

**Review by BlueAcidFrog — CHANGES_REQUESTED** [review]
@zeropaper can you please look into this?

File: src/markdown/blog/oauth2-api-ory-cloud-update.mdx [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+title: OAuth2.0 APIs for Ory Cloud on the horizon
+teaser: OAuth2.0 for Ory Cloud is coming
+subtitle: Leverage OAuth2.0 and OIDC services in directly in Ory Cloud.
+overline: Open Authorization Standards
+publishedAt: 2022-01-21
+published: true
+featuredimage: media/logo-copy.png
+tags: []
+---
+## OAuth2.0 service coming to Ory Cloud
```
Comment (zepatrik) [inline src/markdown/blog/oauth2-api-ory-cloud-update.mdx:14]: Having a title here as well as in the preface results in two titles on the page btw :wink:

File: src/markdown/blog/oauth2-api-ory-cloud-update.mdx [hunk 2]
```diff
@@ -0,0 +1,42 @@
+---
+templateKey: blog
+path: oauth2-api-ory-cloud-update
```
Comment (MicLeey) [inline src/markdown/blog/oauth2-api-ory-cloud-update.mdx:3]:
```suggestion
path: /oauth2-api-ory-cloud-update/
```

Comment (MicLeey) [reply src/markdown/blog/oauth2-api-ory-cloud-update.mdx:3]: This fixes the blog not showing, but if it is autogenerated it should probably be set on the CMS side

**Review by BlueAcidFrog — APPROVED** [review]

**Pull request merged**